src/reports/IdReportGenerator.js

```diff
@@ -0,0 +1,24 @@
+import moment from 'moment';
+import { exportCsv } from '@folio/stripes/util';
+
+class IdReportGenerator {
+  constructor(fileNamePrefix) {
+    this.fileNamePrefix = fileNamePrefix;
+  }
+
+  parse(records, recordFinder = record => record) {
+    return records.map(record => ({ id: recordFinder(record) }));
```

Comment:
Also can simply pass `records.map(record => ({ id: record }))`
If `recordFinder` by default simply returns the record and we don't override this parameter anywhere then we can remove it